usaspending_api/recipient/v2/views/states.py

```diff
@@ -53,8 +53,12 @@ def obtain_state_totals(fips, year=None, award_type_codes=None, subawards=False)
         queryset = (
             matview_search_filter(filters, SummaryStateView)
             .values("pop_state_code")
-            .annotate(total=Sum("generated_pragmatic_obligation"), distinct_awards=StringAgg("distinct_awards", ","))
-            .values("distinct_awards", "pop_state_code", "total")
+            .annotate(
+                total=Sum("generated_pragmatic_obligation"),
+                distinct_awards=StringAgg("distinct_awards", ","),
+                face_value_of_loan=Sum("face_value_loan_guarantee"),
+            )
+            .values("distinct_awards", "pop_state_code", "total", "face_value_loan_guarantee")
```

Comment:
Doesn't this value need to list the annotated field `face_value_of_loan` and not the db column?

Reply:
I made the fields match